documentation/src/main/asciidoc/public/reference/_indexing-plan.adoc

```diff
@@ -113,10 +113,17 @@ include::../components/_writing-reading-intro-note.adoc[]
 
 [WARNING]
 ====
-Indexing synchronization is only relevant when <<coordination-none,coordination is disabled>>.
-
-With coordination strategies such as <<coordination-outbox-polling,`outbox-polling`>>,
-indexing happens in background threads and is always asynchronous.
+No matter if the <<coordination-none,coordination is disabled>> or a coordination strategy such as <<coordination-outbox-polling,`outbox-polling`>>
+is used, when it comes to indexing, Hibernate Search relies on indexing plans.
```

Comment:
Not true for mass indexing though.

Also, it seems to me that the confusion in [the Discourse topic](https://discourse.hibernate.org/t/configuration-property-automatic-indexing-synchronisation-strategy-causes-exceptions-when-used-with-outbox-polling-coordination/7995) was to think that because listener-triggered indexing is disabled, indexing plan configuration does not matter. Which is wrong because indexing plans can still be created explicitly.

So I'm not sure what this sentence brings to the table... ? I'd just remove it TBH.

```suggestion
```